app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt

```diff
@@ -4,13 +4,14 @@ import android.content.Context
 import android.view.View
 import android.view.inputmethod.InputMethodManager
 
-/** KeyboardHelper helps to change the visibility of softinputkeybord. */
+/** KeyboardHelper helps to change the visibility of the soft input keyboard. */
 class KeyboardHelper {
   companion object {
     /**
-     * This method hides softinputkeybord
-     * @param view is the input view
-     * @param context context of the activity
+     * This method hides the soft input keyboard
```

Comment:
```suggestion
     * This method hides the soft input keyboard.
```

Reply:
> 

I addressed all the changes in a new PR.  @BenHenning PTAL